src/ScalarPotentials/ScalarPotential.jl

```diff
@@ -81,24 +83,26 @@ end
 
 function extend_2D_to_3D_by_n_points(sp::ScalarPotential{T, 3, Cylindrical}, axφ::DiscreteAxis{AT, :reflecting, :reflecting}, int::Interval{:closed, :closed, AT},
         n_points_in_φ::Int ) where {T, AT}
-    new_int::Interval{:closed, :open, AT} = Interval{:closed, :open, AT}(0, 2π)
+    #new_int::Interval{:closed, :open, AT} = Interval{:closed, :open, AT}(0, 2π)
     new_ticks::Vector{AT} = Vector{AT}(undef, n_points_in_φ)
     new_pot = Array{eltype(sp.data), 3}(undef, size(sp, 1), n_points_in_φ, size(sp, 3))
     Δφ::AT = 2π / n_points_in_φ
     for i in 1:n_points_in_φ
-        new_ticks[i] = (i - 1) * Δφ
+        new_ticks[i] = mod((i - 1) * Δφ + sp.grid[2][1],T(2π))
```

Comment:
If we start from a `2D` potential, I do not think that we need this modification. We can just use the previous code and have the electric potential range from `[0,2π[`. (Also, I do not see why we would need to include the original `2D` tick value in the `new_ticks` for the `3D` potential here, as everything will be `φ`-symmetric).

Reply:
That is correct, I did this mainly for consistency reasons.